packages/phoenix-evals/src/phoenix/evals/models/openai.py

```diff
@@ -478,6 +478,12 @@ def supports_function_calling(self) -> bool:
             return False
         return True
 
+    @property
+    def _timeout(self) -> Optional[int]:
+        if "o1" in self.model:
+            return 240
```

Comment:
this number seems somewhat arbitrary? you can't set the timeout on the model itself?

Reply:
I would love a timeout on the model itself. o1 can think for longer than 240 seconds with a high reasoning effort.

Reply:
that's a good idea!